### PR TITLE
Ch6 deidentification

### LINKED ISSUE #331: Crossed subjects in chapters 6 and 5
From @mariaruth 

- [x] `iefieldkit`, duplicated entries:
> Content overlap with Ch5 here as well. i think this is useful to have in both places, but we should add a more explicit reference to the other section. Ch5 should focus on data quality monitoring; Ch6 maybe covers how to address corrections? or just notes that this code should be integrated into cleaning do-files?

- [x] de-identification

### PR DESCRIPTION
**Pull request by luizaandrade**
- Resolves #331 
- Addresses #327

### CODE REVIEW

File: chapters/data-analysis.tex [hunk 1]
```diff
@@ -145,36 +145,70 @@ \subsection{De-identification}
...
+simplifies the initial de-identification process.
+If you haven't done that, that are a few tools that can help you with it.
+JPAL's \texttt{PII scan}, as indicated by its name,
+scans variable names and labels for common string patterns associated with identifying information.\sidenote{
+	\url{https://github.com/J-PAL/PII-Scan}}
+The World Bank's \texttt{sdcMicro}
+lists variables that uniquely identify observations,
+as well as allowing for more sophisticated disclosure risk calculations.\sidenote{
+	\url{http://sdctools.github.io/sdcMicro/articles/sdcMicro.html}}
+\texttt{iefielkit}'s \texttt{iecodebook}
```
Comment (bbdaniels) [inline chapters/data-analysis.tex]:
```suggestion
The \texttt{iefieldkit} command \texttt{iecodebook}
```

File: chapters/data-analysis.tex [hunk 2]
```diff
@@ -145,36 +145,70 @@ \subsection{De-identification}
...
+function lists all variables in a data set and exports an Excel sheet
```
Comment (bbdaniels) [inline chapters/data-analysis.tex]:
```suggestion
lists all variables in a data set and exports an Excel sheet
```

File: chapters/data-analysis.tex [hunk 3]
```diff
@@ -145,36 +145,70 @@ \subsection{De-identification}
...
+where you can easily select which variables to keep or drop.\sidenote{
+	\url{https://dimewiki.worldbank.org/wiki/Iecodebook}} 
+
+Once you have a list of variables that contain PII, 
+assess them against the pre-analysis plan and ask:
```
Comment (bbdaniels) [inline chapters/data-analysis.tex]:
```suggestion
assess them against the analysis plan and ask: 
```

**Comment by bbdaniels** [comment]
Good to go except proofing and style checks I think; and the following two corrections:

https://github.com/worldbank/d4di/blob/3999aa027501eacc778aeac16d7c881de599e5e1/chapters/data-analysis.tex#L284

https://github.com/worldbank/d4di/blob/3999aa027501eacc778aeac16d7c881de599e5e1/chapters/data-analysis.tex#L411

File: chapters/data-analysis.tex [hunk 4]
```diff
@@ -193,9 +227,13 @@ \subsection{Correction of data entry errors}
...
-Looking for duplicated entries is usually part of data quality monitoring,
-as is the only other reason to change the raw data during cleaning:
-correcting mistakes in data entry.
+As discussed in the previous chapter,
+looking for duplicated entries is usually part of data quality monitoring,
+and is typically addressed as part of that process.
+So, in practice, you will start writing data cleaning code during data collection.
+The other only other case when changes to the raw data are made during cleaning
+is also directly connected to data quality monitoring:
+it's when you need to correct mistakes in data entry.
```
Comment (kbjarkefur) [inline chapters/data-analysis.tex]: I do not agree. I think this should be in a correction file that is run immediately after the raw data is imported. We need to document changes made to the raw data. The corrected version of the raw data might be more correct, but it is not longer raw.

File: chapters/data-analysis.tex [hunk 5]
```diff
@@ -145,36 +145,70 @@ \subsection{De-identification}
...
+	\url{http://sdctools.github.io/sdcMicro/articles/sdcMicro.html}}
+The \texttt{iefieldkit} command \texttt{iecodebook}
+lists all variables in a data set and exports an Excel sheet
+where you can easily select which variables to keep or drop.\sidenote{
+	\url{https://dimewiki.worldbank.org/wiki/Iecodebook}} 
+
+Once you have a list of variables that contain PII, 
+assess them against the analysis plan and ask: 
+will this variable be needed for analysis?
+If not, the variable should be dropped.
```
Comment (kbjarkefur) [inline chapters/data-analysis.tex]:
```suggestion
If not, the variable should be dropped. Don't be afraid to drop too many variables the first time, as you can always go back and remove variables from the list of variables to be dropped, but you can not go back in time and drop a PII variable that was leaked as it was incorrectly kept
```

File: chapters/data-analysis.tex [hunk 6]
```diff
@@ -145,36 +145,70 @@ \subsection{De-identification}
...
+If not, the variable should be dropped. 
+Examples include respondent names, enumerator names, interview date, respondent phone number.
+If the variable is needed for analysis, ask: 
+can I encode or otherwise construct a variable to use for the analysis that masks the PII, 
+and drop the original variable?
+This is typically the case for most identifying information.
+Examples include geocoordinates 
+(after constructing measures of distance or area, 
+drop the specific location), 
+and names for social network analysis (can be encoded to unique numeric IDs).
```
Comment (kbjarkefur) [inline chapters/data-analysis.tex]:
```suggestion
and names for social network analysis (can be encoded to secret and unique IDs).
```

Doesn't have to be numeric, and it is important they are kept secret.

File: chapters/data-analysis.tex (hunk 4 above)
Comment (kbjarkefur) [reply chapters/data-analysis.tex]: Say data values or something, not `raw data` as it too associated with raw data as in the most original version of the data set

Comment (luizaandrade) [reply chapters/data-analysis.tex]: done in 50e1d81

**Comment by luizaandrade** [comment]
342e476 creates the separate section for deidentification that Maria requested.

**Pull request merged**